www/student.js

```diff
@@ -86,3 +87,33 @@ endButton.addEventListener('click', function () {
   runButton.disabled = false
   endButton.disabled = true
 })
+
+// Timer functionality
+let timer_running = true
+let minute = 0
+let second = 0
+
+function timer() {
+
+  if (timer_running) second++
+  if (second == 60) {
+    minute++
+    second = 0
+  }
+
+  let minString = minute
+  let secString = second
+
+  if (minute < 10) {
+    minString = '0' + minString
+  }
+  if (second < 10) {
+    secString = '0' + secString
+  }
+
+  document.querySelector('#min').innerHTML = minString
+  document.querySelector('#sec').innerHTML = secString
```

Comment:
I think there might be a simpler way to do this. As far as I can see, the timer could be displayed as an hh:mm:ss string in a single element. In that case, the text can be set using the following:

```
timerValue = new Date(1000 * seconds).toISOString().substr(11, 8)
```